src/Compilers/CSharp/Portable/BoundTree/BoundTreeVisitors.cs

```diff
@@ -185,6 +185,7 @@ public static Location GetTooLongOrComplexExpressionErrorLocation(BoundNode node
         /// <summary>
         /// Consumers must provide implementation for <see cref="VisitExpressionWithoutStackGuard"/>.
         /// </summary>
+        [DebuggerStepThrough]
```

Comment:
🙌I have a patchset that applies this same change locally to my source. Great to see it in the actual source